Source/RSP/dma.c

```diff
@@ -39,7 +39,7 @@ void SP_DMA_READ (void)
 
     addr = (*RSPInfo.SP_DRAM_ADDR_REG) & 0x00FFFFFF;
 
-	if (addr > 0x800000)
+	if (addr > 0x7FFFFF)

```

Comment:
The alternative solution:
```c
if (addr >= 0x800000UL)
```
as without the "UL" at the end, a 24-bit signed int type could equate that to
```c
if (addr > INT_MIN)
```
which would have caused the MessageBox 99% of the time and have been a bug, so I like 7FFFFF.